txpool/txpool.go

```diff
@@ -473,12 +489,17 @@ func (t *TxPool) ProcessEvent(evnt *blockchain.Event) {
 
 	// remove the mined transactions from the pendingQueue list
 	for _, txn := range delTxns {
-		t.decreaseSlots(numSlots(txn))
+		t.decreaseSlots(slots(txn))
 		t.pendingQueue.Delete(txn)
 		t.remoteTxns.Delete(txn)
 	}
 }
 
+// Returns the number of slots currently occupying the pool
```

Comment:
You don't need this method. Use gauge.height directly or rename it.

Reply:
I guess it's more a matter of what we'd like to export from a package (since the same could be said for [(*Txpool) length()](https://github.com/0xPolygon/polygon-sdk/blob/develop/txpool/txpool.go).

Maybe I'm going about this wrong, but when writing unit tests I feel like we should be testing API calls that the package is providing (much like the rest of the code is importing `TxPool` and using it). This could the part that makes  tests somewhat unstandardised and harder to "read".

Reply:
This method is not exported anywhere, that's why @brkomir is suggesting we just get rid of it

Reply:
[Removed](https://github.com/0xPolygon/polygon-sdk/pull/213/commits/97b98b295063247033647f8cfca2d7dd5b82378a)